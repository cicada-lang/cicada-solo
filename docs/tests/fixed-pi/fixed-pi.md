# vector_null

``` cicada
function vector_null(fixed E: Type): Vector(E, zero) {
  return vecnil
}

vector_null
```
