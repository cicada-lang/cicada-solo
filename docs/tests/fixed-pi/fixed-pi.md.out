(fixed E) => vecnil: (fixed E: Type) -> Vector(E, 0)